build/yamls/antrea-windows-containerd-with-ovs.yml

```diff
@@ -5,9 +5,29 @@ data:
     mkdir -force c:/var/log/antrea
     $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
     $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/')
-    mkdir -force C:/var/run/secrets/kubernetes.io/serviceaccount
-    cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/ca.crt C:/var/run/secrets/kubernetes.io/serviceaccount
-    cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/token C:/var/run/secrets/kubernetes.io/serviceaccount
+
+    #From containerd version 1.7 onwards servcieaccount directory and the ca.cert and token files will automatically be created.
```

Comment:
```suggestion
    # From containerd version 1.7 onwards, the servcieaccount directory, the ca.cert and token files will
    automatically be created.
```

Reply:
I suppose these two Windows manifests are generated automatically? Why does this one include this comment but not in another file?